pkg/reader/generic/sql.go

```diff
@@ -29,7 +32,7 @@ func (s *SqlReader) GetColumns(table string) (columns []string, err error) {
 	}
 
 	for k, column := range columns {
-		columns[k] = fmt.Sprintf("%s.%s", table, column)
```

Comment:
This will be necessary for the relationships

Reply:
In that case we also need to change https://github.com/hellofresh/klepto/pull/52/files#diff-f7a183c94b4fefc7a5450142e7ef915aL88 to have the same format if you want to do that